base/src/main/scala/co/uproot/abandon/Report.scala

```diff
@@ -252,6 +252,9 @@ object Reports {
 
   def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings): xml.Node = {
     <abandon>
+      <version>
+        { BuildInfo.version }
+      </version>
```

Comment:
> I didn't realize that the spaces in the curly braces are actual spaces in the generated xml.

They are not; you can keep *them*. I thought you were referring to spaces between the XML tag and the scala code; those are significant; hence need to be removed.

Sorry for the confusion.

:+1: for attribute.